gcp/modules/gcp-stackdriver-monitoring/alert_couchdb_cluster_missing_node.tf

```diff
@@ -0,0 +1,35 @@
+resource "google_monitoring_alert_policy" "couchdb_missing_node" {
+  display_name = "CouchDB nodes are visible by the cluster"
+  combiner     = "OR"
+
+  conditions {
+    condition_threshold {
+      filter          = "metric.type=\"logging.googleapis.com/user/couchdb_missing_node.error\" resource.type=\"k8s_container\""
+      comparison      = "COMPARISON_GT"
+      threshold_value = 0
+      duration        = "600s"
+
+      aggregations {
+        alignment_period   = "60s"
+        per_series_aligner = "ALIGN_SUM"
+        group_by_fields    = []
```

Comment:
I think this is not needed, and won't have any effect unless `cross_series_reducer` is udes.

Reply:
came from C&P from other alerts, I deleted all of them in all alerts.